src/reports/components/fast-pass-report-summary.tsx

```diff
@@ -25,12 +28,15 @@ export class FastPassReportSummary extends React.Component<FastPassReportSummary
         const failedTabResults = [];
         const incompleteTabResults = [];
         const passedTabResults = [];
+        const displayAutmatedInfo =
```

Comment:
nit: spelling error, same as in `tab-stops-failed-instance-section.tsx`, but not a big deal since this is temporary until we remove the feature flag